business/jaeger_helper.go

```diff
@@ -0,0 +1,64 @@
+package business
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/kiali/kiali/config"
+)
+
+type Trace struct {
+	Id string `json:"traceID"`
+}
+
+type RequestTrace struct {
+	Traces []Trace `json:"data"`
+}
+
+// ConfigToJS generates env.js file from Kiali config
```

Comment:
Perhaps, could you remove this comment?
It confused me. I was trying to find how this func is related to the env.js file, but I don't manage to find any relation. So, if it's not related, please remove to avoid confusion if future code changes are needed ;)

Reply:
Yes I change a lot this PR and I forgot remove it anyway I am still working on this PR so please don't review it while DNM label is in the PR to don't waste time.